Jenkinsfile

```diff
@@ -51,4 +50,9 @@ runTheBuilds.runDevToolsProject(
         '8000', '-v jenkins-nodes:/jenkins_nodes', env.CONTAINER_ARGS)
     }
   },
+  cleanup: { data ->
+    if (data?.venvs) {
```

Comment:
This must be `data?.venv` (same for the line below). We don't use a `VirtualEnvList` in this Jenkinsfile.

Reply:
Addressed in 372875a.